### ISSUE TITLE
[Bug]: Move Undesigned Components to the Demo Application

### ISSUE THREAD

**Issue by Tayebsed93** [Orange-OpenSource/ouds-flutter#184]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

iPhone 16 plus

### OS Version

18.4

### App version

v0.3.0

### Describe the issue

It has been observed that several undesigned components from the ouds_core library are currently present in the library. To improve project clarity and maintainability, these components should be moved to the demo application. Once they are designed, they can be reintegrated into the ouds_core library.

### Destination Folder
Place the undesigned components in the following folder of your demo application:
`app/lib/undesign_components/`

**Comment by B3nz01d** [comment]
Completed during the sprint 18
